ctms/bin/acoustic_sync.py

```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env python3
 """Run continuously in the background, syncing acoustic with our db."""
-
 from time import monotonic, sleep
 
 from ctms import config
 from ctms.database import get_db_engine
+from ctms.sync import CTMSToAcousticSync
 
 
 def main(db, settings):
+    sync_service = CTMSToAcousticSync(
```

Comment:
So this is where my acoustic knowledge breaks down:

We get a refresh token from them that doesn't change. Does the pysilverpop client handle refreshing our creds in the background or will this one we construct outside of the loop eventually go out of date?

Reply:
From my understanding (imparted by @cvalaas, so correct me if I'm wrong!)
... is that the token is not expected to change and should not need to be updated.

Reply:
They use [OAuth2Session](https://github.com/theatlantic/pysilverpop/blob/ee9d60a6e99d36f123729746ceba6003ac56cb59/silverpop/api.py#L179-L183), in a similar way to Basket when it calls CTMS. I've read that code, the class takes care of storing any changed refresh token.

Reply:
and, like CTMS, the refresh URL is also the token URL, and you pass in the client ID and secret, so I'm guessing Acoustic's token granting implementation is very close to CTMS's, and the refresh token is nonsense.

Reply:
I know the refresh token is required, but if it changes (I expect it does during a session), then I would expect pysilverpop to handle that.